supervisord.conf

```diff
@@ -0,0 +1,42 @@
+; Sample supervisor config file.
+;
+; For more information on the config file, please see:
+; http://supervisord.org/configuration.html
+;
+; Notes:
+;  - Shell expansion ("~" or "$HOME") is not supported.  Environment
+;    variables can be expanded using this syntax: "%(ENV_HOME)s".
+;  - Comments must have a leading space: "a=b ;comment" not "a=b;comment".
+
+[unix_http_server]
+file=/tmp/supervisor.sock  ; (the path to the socket file)
+
+[inet_http_server]         ; inet (TCP) server disabled by default
+port=127.0.0.1:9001        ; (ip_address:port specifier, *:port for all iface)
+username=admin             ; (default is no username (open server))
+password=admin             ; (default is no password (open server))
```

Comment:
do we want the password to live here? I mean anyone can then go to the VM's public_ip/9001, log in and stop all the processes. Alternatively, I think the password should be updated by the admin at the time of deployment.